content/applications/inventory_and_mrp/inventory/management/misc/cluster_picking.rst

```diff
@@ -0,0 +1,221 @@
+   :alt: Activate *Packages* and *Batch Transfers* features in the settings.
+
+Since batch picking is a method to optimize the *pick* operation in Odoo, the :guilabel:`Storage
+Locations` and :guilabel:`Multi-Step Routes` options under the :guilabel:`Warehouse` heading must
+also be checked on this settings page. When finished, click :guilabel:`Save`.
+
+.. image:: cluster_picking/locations-routes-checkbox.png
+   :align: center
+   :alt: Enable *Storage Locations* and *Multi-Step Routes* Inventory > Configuration > Settings.
+
+.. _inventory/misc/create-package:
+
+Packages setup
+--------------
+
+After the :guilabel:`Packages` feature is enabled, set up the packages before the picking, in the
+:menuselection:`Inventory app --> Products --> Packages` page. Click the :guilabel:`New` button to
+create a new package.
+
+On the package form for creating a new package, the :guilabel:`Package Reference` is pre-filled to
+the next available `PACK` number in the system. :guilabel:`Pack Date` is automatically set to the
+creation date of the form.
+
+For the :guilabel:`Package Use` field, the :guilabel:`Disposable Box` radio option should be
+selected if the package is a shipping container. On the contrary, the :guilabel:`Reusable Box`
+option should be selected if the package is simply used as a method of grouping products from the
+same |SO| together and the products are intended to be moved to the intended shipping container at
+the output location.
+
+.. seealso::
+   :ref:`Packages <inventory/management/packages>`
+
+.. example::
+   A package intended for cluster picking is named `CLUSTER-PACK-3` for easy identification. For
+   this workflow, the products are directly packed using their intended shipping containers, so
+   :guilabel:`Package Use` is set to :guilabel:`Disposable Box`.
+
+   .. image:: cluster_picking/cluster-package.png
+      :align: center
+      :alt: Create new package form.
+
+Create cluster batch
+====================
+
+To see how cluster picking works in Odoo, navigate to the *Sales* app and create |SOS| that will be
+fulfilled together in the same batch. After confirming the |SO|, the :guilabel:`Delivery` smart
+button becomes visible. Displayed inside the icon is a number representing the amount of steps in
+the outgoing shipment process.
+
+.. example::
+   Begin by creating three |SOS| for the apples, oranges, and bananas as shown in the :ref:`example
+   above <inventory/misc/cluster_picking/example>`.
+
+   After confirming the |SO|, the :guilabel:`Delivery` smart button will be shown with the number
+   `2`, indicating that there are two operations to complete: `Pick` and `Delivery`.
+
+   .. image:: cluster_picking/create-sales-order.png
+      :align: center
+      :alt: Example sales order for an apple, orange, and banana.
+
+With the |SOS| created, orders now must be grouped into batches. To do so, navigate to the
+*Inventory* dashboard and select the operation type card, `Delivery Orders` or `Pick`. Doing so
```

Comment:
Wouldn't you always select the Pick card, since that operation needs to be completed before processing the delivery order?

Reply:
Thanks for bringing this up because I was thinking about this too.

It's complicated because it's not necessary to have 2-step delivery to use a picking method. (The Pick operation type is enabled when we select 2-step)

Does this line not make sense when I write it this way? Do let me know, and I'll think about how to rewrite it!